index.html

```diff
@@ -2,9 +2,9 @@
 <html>
     <head>
         <meta charset="utf-8" />
-        <meta http-equiv="refresh" content="0; url=https://join.slack.com/t/crossplane/shared_invite/zt-26kxm0k2a-esMwXvMpfw8glyoMzukAlQ" />
+        <meta http-equiv="refresh" content="0; url=https://join.slack.com/t/crossplane/shared_invite/zt-29oiqovki-WuHqY~LUaAIHZVChxJNUqg" />
     </head>
     <body>
         <p><a href="https://join.slack.com/t/crossplane/shared_invite/zt-26kxm0k2a-esMwXvMpfw8glyoMzukAlQ">Redirecting you to a Crossplane Slack invitation</a></p>
```

Comment:
The link should to be updated here too for completeness ☝️ 

```suggestion
        <p><a href="https://join.slack.com/t/crossplane/shared_invite/zt-29oiqovki-WuHqY~LUaAIHZVChxJNUqg">Redirecting you to a Crossplane Slack invitation</a></p>
```

feel free to make this change yourself instead of accepting this suggestion - suggestions in the commit history can be annoying sometimes 🤷